bitnami/opensearch/values.yaml

```diff
@@ -3405,3 +3405,181 @@ dashboards:
     ## @param dashboards.persistence.dataSource Custom PVC data source
     ##
     dataSource: {}
+
+## @section OpenSearch Snapshots Parameters
+snapshots:
+  ## @param snapshots.command Override default container command (useful when using custom images)
+  ##
+  command:
+    - /usr/bin/curl
+  ## Configure Container Security Context
+  ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/security-context/#set-the-security-context-for-a-pod
+  ## @param snapshots.containerSecurityContext.enabled Enabled containers' Security Context
+  ## @param snapshots.containerSecurityContext.seLinuxOptions [object,nullable] Set SELinux options in container
+  ## @param snapshots.containerSecurityContext.runAsUser Set containers' Security Context runAsUser
+  ## @param snapshots.containerSecurityContext.runAsGroup Set containers' Security Context runAsGroup
+  ## @param snapshots.containerSecurityContext.runAsNonRoot Set container's Security Context runAsNonRoot
+  ## @param snapshots.containerSecurityContext.privileged Set container's Security Context privileged
+  ## @param snapshots.containerSecurityContext.readOnlyRootFilesystem Set container's Security Context readOnlyRootFilesystem
+  ## @param snapshots.containerSecurityContext.allowPrivilegeEscalation Set container's Security Context allowPrivilegeEscalation
+  ## @param snapshots.containerSecurityContext.capabilities.drop List of capabilities to be dropped
+  ## @param snapshots.containerSecurityContext.seccompProfile.type Set container's Security Context seccomp profile
+  ##
+  containerSecurityContext:
+    enabled: true
+    seLinuxOptions: null
+    runAsUser: 1001
+    runAsGroup: 1001
+    runAsNonRoot: true
+    privileged: false
+    readOnlyRootFilesystem: true
+    allowPrivilegeEscalation: false
+    capabilities:
+      drop: ["ALL"]
+    seccompProfile:
+      type: "RuntimeDefault"
+  ## @param snapshots.enabled Enable automatic setup of repositories and snapshot policies
+  ##
+  enabled: false
```

Comment:
I'd move this to the beginning of the section to give more visibility

Reply:
Sure. I sorted it alphabetically, but visibility makes sense. I'll move it.

Reply:
Should `snapshots.persistence.enabled` be moved to the top of `snpashots.persistence`, so they are similar?

Reply:
I've moved both to the top. If you don't want the same for persistence, I'll just revert the commit for you. Please mark this conversation as resolved if you are happy with the solution!